simpleviewinc/appetite
# Appetize Wrapper

# Goals
* implement an appetize api wrapper (starting with the upload endpoint)

# Updates
* I simply refactored and wrote tests for the appetize wrapper that was based in tap-snack before

# Testing
* Clone the repo
* Create a `.env` file at the root directory
* Fill in this:
```bash
PLATFORM=ios
TOKEN=<your appetize token>
URL=https://exp-shell-app-assets.s3.us-west-1.amazonaws.com/ios/%40mpcarolin/kegcore-4ed71545-f25d-4563-83a0-4e05c302262c-simulator.tar.gz
# BUILD_PATH=<path to a local build file>
```
* Run `yarn test` -- ensure all tests pass
* Run `yarn apt:test`
  * ensure the request succeeds
* Now comment out URL in your `.env` and uncomment `BUILD_PATH`
* Run `yarn apt:test` again and verify that succeeds
